### PR TITLE
Disable the Storage Metrics emmiter for now.

### LINKED ISSUE #5468: Remove the storage reporting for domains
We recently had a showstopping SQL CPU utilization issue with the domain reporting. This was reported [here](https://github.com/pulp/pulpcore/issues/5442) and an attempt to fix it was [made here](https://github.com/pulp/pulpcore/pull/5441). Then another attempt was [made here](https://github.com/pulp/pulpcore/pull/5453).

We can't re-enable the OTEL metrics in our environment until this is resolved. Also we don't have capacity to resolve this in a timely manner.

So given ^ perspectives, I am requesting the feature be removed until we can fix it adequately and re-enable it.

### PR DESCRIPTION
**Pull request by decko**
Closes #5468

### CODE REVIEW

**Review by lubosmj — COMMENTED** [review]
To be on the safe side, we will need to disable https://github.com/pulp/pulpcore/blob/9e9198ca115d14229c85a4db4da1d811846e7462/pulpcore/app/models/domain.py#L78 too. This is a hook for newly created domains.

It is so unfortunate that we could not validate this patch: https://github.com/pulp/pulpcore/pull/5452.

**Comment by decko** [comment]
> It is so unfortunate that we could not validate this patch: #5452.

I think we should accept and set it as validated. But yet it won't solve our issue.

File: CHANGES/5468.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Disable the Storage Metrics Emitter until we figure its resource usage.
```
Comment (lubosmj) [inline CHANGES/5468.misc:1]:
```suggestion
Temporarily disabled domains' disk usage reporting.
```

**Review by lubosmj — APPROVED** [review]
Left a non-blocking comment.

**Review by lubosmj — APPROVED** [review]

**Review by dkliban — APPROVED** [review]

File: CHANGES/5468.misc [hunk 2]
```diff
@@ -0,0 +1 @@
+Temporarily disable domains' disk usage reporting.
```
Comment (daviddavis) [inline CHANGES/5468.misc:1]: Is misc the right type of entry?

Comment (decko) [reply CHANGES/5468.misc:1]: I believe so @daviddavis. We didn't fixed it. We're just disabling and will get back later to it.

**Pull request merged**